extensions/amp-subscriptions/0.1/amp-subscription-platform.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Services} from '../../../src/services';
+
+/**
+ * This implements the methods to interact with various subscription platforms.
+ * @implements {./subscription-platform}
+ */
+export class AmpSubscriptionPlatform {
```

Comment:
Can we rename to `LocalSubscriptionPlatform`? And also, this file needs a dedicated test file.